src/components/DpPager/DpPager.vue

```diff
@@ -79,7 +80,15 @@ export default {
 
   data () {
     return {
-      itemsPerPage: this.perPage <= this.totalItems ? this.perPage : this.totalItems
+      itemsPerPage: this.perPage <= this.totalItems ? this.perPage : this.totalItems,
+      translations: {
+        multipleOf: de.pager.multipleOf,
+        multipleItems: de.pager.multipleItems,
+        selectNumberOfItems: de.pager.selectNumberOfItems({
+          results: this.totalItems,
+          items: this.multipleItems
```

Comment:
Did you test this? Should it not be `this.translations.multipleItems`?

Reply:
You are right, thank you
Done in https://github.com/demos-europe/demosplan-ui/pull/918/commits/3596f75713f6a155195aa2c0c1ec4926e3d0ffc6

Reply:
I believe you linked the wrong commit?

Reply:
No, I think it's the right commit!

Reply:
Right, I must have opened the wrong tab, there are just too many of them..